docs/docs/docs/developer-resources/testing.md

```
@@ -1,10 +1,250 @@
+## Database Management
+
+This section covers easy ways for developers to validate their work
+
+### CloudBeaver
+
+CloudBeaver is a lightweight web application designed for comprehensive data management. It allows you to work with various data sources, including SQL, NoSQL, and cloud databases, all through a single secure cloud solution accessible via a browser.
+
+#### Accessing the PostgreSQL Database using CloudBeaver
+
+1. Open your preferred browser and navigate to:
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)